examples/framesjs-starter/app/examples/new-api-custom-font/frames/route.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/jsx-key */
+import { frames } from "./frames";
+
+// without this line, this type of importing fonts doesn't work for some reason
+export const runtime = "edge";
+
+const regularFont = fetch(
+  new URL("/public/Inter-Regular.ttf", import.meta.url)
+).then((res) => res.arrayBuffer());
```

Comment:
It took me several attempts to figure this out, but for the nodejs runtime deployed to Vercel, it seems like the most reliable way was to use a combination of `process.cwd()` and `fs` functions:

https://github.com/chainpatrol/phishframe/blob/0fba55e76536ad57e12815c3b90ff0723124bc0a/src/app/frames/route.tsx#L147-L152

Reply:
dang thats tricky - thanks for following up. Will incorporate